### DISCUSSION TITLE
Welcome to Emulair Discussions!

### DISCUSSION THREAD

**Discussion by RaduBratan** [EmulairEmulator/Emulair-Lemuroid-clone#42] [Announcements]
## 👋 Welcome!
I'm using Discussions as a place to connect with other members of this community. I hope that you:
  * Ask questions you’re wondering about (on the Q&A page).
  * Share ideas (on the Ideas page).
  * Engage with other community members (on any page).
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by LittleFang-Coyote** [comment]
Hello! Sorry, I do not know how to use github, I decided to create an account just to post a comment here, so please forgive my lack of etiquette.

I saw that this is a passion project of yours? Reading another discussion, I see you work on this project on your free time meanwhile you balance your work and studies life. I do not have no kind of skill to help you with this project of yours, but I figure it out I could at least leave a word of encouragement/apreciation, as an "artist" myself I know how some words can give a boost of motivation!

Even thought I do not know nothing about codding or github itself, I see that you have a really good organization and some impresive goals. You conviced me, a random user, to belive in this project! As yourself said "I doubted this project would ever gain any traction." things come with time, do it for yourself and soon it will reach its target audience, and I sure that if you dont let this project slide into the forgotten pitch it will shiny as a great emulator. You doing great, keep going.

Wish I could help more then words, but I hope this comment at least made you smile, good luck with your studies, job and projects, don't back down just yet. Have a good day!

**Reply by RaduBratan** [reply]
Omg, thank you so much! I appreciate your words of encouragement, I'll remember them next time I feel less motivated to work on Emulair :))))

I'll also give you some hope in return. The end of my final semester is getting closer and closer, and after I'll finish my exams and the Bachelor's Thesis + project (which, guess what, is *another* Android app), I'll immediately start working on Emulair, having obtained all the app development experience I dreamed about 1.5 years ago when I wrote my first line of code for Emulair.

And you too have a good day!

**Reply by RaduBratan** [reply]
Also, yes, you came to the correct place for discussions. The Issues page is mostly used for requesting changes/reporting bugs/general development stuff.
